src/Transaction.php

```diff
@@ -333,6 +333,32 @@ public static function details(
         return new Result\Details($result);
     }
 
+
+    /**
+     * Gets the services
+     *  
+     * @param string|null $paymentMethodId
+     *
+     * @return Result\GetService
+     * @throws Error\Api
+     * @throws Error\Error
+     * @throws Error\Required\ApiToken    
+     */
+    public static function getService(        
```

Comment:
Kun je deze toelichten?
Waarom heb je deze toegevoegd? Het lijkt me niet logisch om getService aan te roepen van een transaction, deze zou dan ook in de paymentmethods moeten. Ik denk dat deze gewoon weggelaten kan worden, tenzij jij natuurlijk goede argumenten hebt om dat niet te doen.

Reply:
Klopt was idd niet nodig